presentation/common/src/test/_helpers/Content.ts

```diff
@@ -1,148 +1,148 @@
-  autoExpand?: boolean;
-  editor?: EditorDescription;
-  renderer?: RendererDescription;
-  relationshipMeaning?: RelationshipMeaning;
-}) {
-  const nestedContentFieldType: StructTypeDescription = {
-    valueFormat: PropertyValueFormat.Struct,
-    typeName: "NestedContentFieldType",
-    members: props.nestedFields.map((f) => ({
-      name: f.name,
-      label: f.label,
-      type: f.type,
-    })),
-  };
-  const field = new NestedContentField(
-    props.category ?? createTestCategoryDescription(),
-    props.name ?? "NestedContentField",
-    props.label ?? "Nested Content",
-    nestedContentFieldType,
-    props.isReadonly ?? false,
-    props.priority ?? 0,
-    props.contentClassInfo ?? createTestECClassInfo(),
-    props.pathToPrimaryClass ?? createTestRelationshipPath(1),
-    props.nestedFields,
-    props.editor,
-    !!props.autoExpand,
-    props.renderer,
-  );
-  if (props.relationshipMeaning)
-    field.relationshipMeaning = props.relationshipMeaning;
-  field.rebuildParentship();
-  return field;
-}
-
-export function createTestContentDescriptor(props: Partial<DescriptorSource> & { fields: Field[] }) {
-  return new Descriptor({
-    connectionId: "",
-    displayType: "",
-    contentFlags: 0,
-    selectClasses: [createTestSelectClassInfo()],
-    categories: [createTestCategoryDescription()],
-    ...props,
-  });
-}
-
-export function createTestContentItem(props: {
-  primaryKeys?: InstanceKey[];
-  label?: LabelDefinition | string;
-  imageId?: string;
-  classInfo?: ClassInfo;
-  values: ValuesMap;
-  displayValues: DisplayValuesMap;
-  mergedFieldNames?: string[];
-  extendedData?: { [key: string]: any };
-}) {
-  return new Item(
-    props.primaryKeys ?? [createTestECInstanceKey()],
-    props.label ?? "",
-    props.imageId ?? "",
-    props.classInfo,
-    props.values,
-    props.displayValues,
-    props.mergedFieldNames ?? [],
-    props.extendedData,
-  );
-}
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import {
+  CategoryDescription, ClassInfo, Descriptor, DescriptorSource, DisplayValuesMap, EditorDescription, Field, InstanceKey, Item, LabelDefinition,
+  NestedContentField, PropertiesField, Property, PropertyValueFormat, RelationshipPath, RendererDescription, SelectClassInfo, StructTypeDescription,
+  TypeDescription, ValuesMap,
+} from "../../presentation-common";
+import { RelationshipMeaning } from "../../presentation-common/rules/content/modifiers/RelatedPropertiesSpecification";
+import { createTestECClassInfo, createTestECInstanceKey, createTestRelationshipPath } from "./EC";
+
+export const createTestCategoryDescription = (props?: Partial<CategoryDescription>) => ({
```

Comment:
Should exports in this file be tagged `@internal`?

Reply:
done